website/src/pages/product/code-learning.tsx

```diff
@@ -0,0 +1,212 @@
+                </div>
+                <div className="row justify-content-center py-4">
+                    <div className="col-lg-10 text-center">
+                        <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                            <img src="/external-logos/convoy-logo.svg" />
+                        </a>
+                        <blockquote className="blockquote case-studies__quote case-studies__quote--in-content">
+                            <p>
+                                For our new developers, Sourcegraph has been invaluable to get to know the repository
+                                structure, to track down where code lives, and self-service during their investigations.
+                            </p>
+                            <footer className="blockquote-footer">Owen Kim, Senior Software Engineer, Convoy</footer>
+                        </blockquote>
+                    </div>
+                    <div className="col-lg-10 mt-2 py-4 text-center">
+                        <h5>
+                            Learn more about how{' '}
+                            <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                                Convoy uses Sourcegraph to onboard new developers.
+                            </a>
+                        </h5>
+                    </div>
+                </div>
+            </ContentSection>
+            <ContentSection color="white" className="pb-2">
+                <h2 id="features" className="display-4 mb-5 text-center">
+                    Why learning code is faster with Sourcegraph
+                </h2>
+                <p className="text-center mb-5">
+                    Sourcegraph <a href="/universal-code-search">Universal Code Search</a> lets you explore across your
+                    code and public codebases with contextual code intelligence to accelerate understanding code when
+                    onboarding new developers and reviewing unfamiliar code.
+                </p>
+                <div className="row">
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={GlobeIcon} color="green">
+                            <h4>Find definitions and explore references, even across projects</h4>
+                            <p>
+                                See a call to a function you don't recognize? With Sourcegraph, you can see its
+                                documentation and type signature by hovering over it &mdash; and go to its definition in
+                                a single click, across package, dependency, and repository boundaries. Even if your IDE
+                                can do this, Sourcegraph lets you explore code intelligently in your web browser in any
+                                repository on any branch, instantly and without losing your local context to massively
```

Comment:
"to massively save time" sounds awkward coming at the end. It is also hard to parse - because "losing your local context to NOUN" is a common phrase, but this "to massively save time" applies to "explore code".